channel/src/main/java/com/synopsys/integration/alert/channel/msteams2/MSTeamsChannelMessageSender.java

```diff
@@ -62,43 +61,34 @@ public MessageResult sendMessages(MSTeamsJobDetailsModel msTeamsJobDetailsModel,
         Map<String, String> requestHeaders = new HashMap<>();
         requestHeaders.put("Content-Type", "application/json");
 
-        String provider = createProviderString(channelMessages);
-        String messageTitle = String.format("Received a message from %s", provider);
-        Request request = createRequestsForMessage(webhook, messageTitle, channelMessages, requestHeaders);
+        List<Request> messageRequests = channelMessages.stream()
+                                            .map(it -> createRequestsForMessage(webhook, it, requestHeaders))
+                                            .collect(Collectors.toList());
 
-        restChannelUtility.sendSingleMessage(request, msTeamsKey.getUniversalKey());
+        restChannelUtility.sendMessage(messageRequests, msTeamsKey.getUniversalKey());
 
         return new MessageResult(String.format("Successfully sent %d MSTeams message(s)", channelMessages.size()));
     }
 
-    private String createProviderString(List<MSTeamsChannelMessageModel> channelMessages) {
-        return channelMessages.stream()
-                   .findFirst()
-                   .map(MSTeamsChannelMessageModel::getProviderDetails)
-                   .map(ProviderDetails::getProvider)
-                   .map(LinkableItem::getValue)
-                   .orElse("BlackDuck");
-    }
-
-    private Request createRequestsForMessage(String webhook, String messageTitle, List<MSTeamsChannelMessageModel> messages, Map<String, String> requestHeaders) {
-        String jsonString = createJsonString(messageTitle, messages);
+    private Request createRequestsForMessage(String webhook, MSTeamsChannelMessageModel messageModel, Map<String, String> requestHeaders) {
```

Comment:
Should this method be changed from plural to singular? Since it is only returning a single request?